src/Registry/Registry.php

```diff
@@ -57,6 +57,17 @@ public function set(string $name, callable $callback, bool $fresh = false): self
         return $this;
     }
 
+    /**
+     * Check if connection exists
+     *
+     * @param string $name
+     * @return bool
+     */
+    public function has(string $name)
+    {
+        return \array_key_exists($name, $this->callbacks);
+    }
+
```

Comment:
Is this supposed to be here? I think it's causing tests to fail.

Reply:
removed 👍